vtex/admin-ui
# Sidebar design & behavior improvements

#### What is the purpose of this pull request?


Apply design improvements on the Sidebar component based on VTEX client's feedback

List of improvements:

* Apply spacing changes over the section and section items
* Change the sidebar behavior to show a preview of each section items when hovering over the section's icon

#### How should this be manually tested?


1. Check that the spacing of the section items changes between the old version and the new one
2. Hover the icons using your mouse to check the section preview
3. Hover with your mouse on the last section and see that the URL preview doesn't override the icon

#### Screenshots or example usage


> 💡 You can type "a", "s", and "t" to hide storybook gadgets.

**Before**

You can check this [storybook workspace](https://leftnav-before.surge.sh/?path=/story/admin-ui-sidebar--playground) for a preview without the changes applied.

![2022-03-08 15-47-18 2022-03-08 15_47_57](https://user-images.githubusercontent.com/20579226/157305027-179a8281-1a04-493d-8271-4e8187018024.gif)

**After**

You can check the workspace preview on the storybook URL generated by our CI

![leftnav-hover](https://user-images.githubusercontent.com/20579226/157305259-4156c03d-44fa-4a8b-801b-2b878a42e955.gif)

#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
